docs/DeveloperGuide.md

```diff
@@ -1149,14 +1152,154 @@ _
+   
+   2. Test case: `get /wn hello`<br>
+      Expected: No patients are listed. Invalid ward number will never match.
+   
+   3. Test case: `get /wn D31`<br>
+      Expected: No patients are listed. Invalid ward number will never match.
+
+### Filtering patients by floor number
+
+1. Displaying all patients who are on a specific floor
+
+   1. Prerequisite: At least one patient is inpatient and is on the specific floor.
+   
+   2. Test case: `get /fn 3`<br>
+      Expected: All patients who are in floor number 3 are listed. The number of patients listed is displayed in the result box.
+   
+   3. Test case: `get /fn 3 4 5`<br>
+      Expected: All patients who are in floor numbers 3, 4 and 5 are listed. The number of patients listed is displayed in the result box.
+
+   4. Test case: `get floor 3`<br>
+       Expected: The current list remains unchanged. Error message is displayed in the result box.
+
+   5. Test case: `get /fn 0`<br>
+      Expected: No patients are listed. Error message is displayed in the result box.
+   
+   6. Test case: `get /fn -1 4 5`<br>
+      Expected: No patients are listed. Error message is displayed in the result box.
+
+   7. Test case: `get /fn -1 hello -5`<br>
+      Expected: No patients are listed. Error message is displayed in the result box.
+
+## **Appendix: Effort**
+
+### Difficulty level
+
+Overall difficulty was manageable, as many of our iP implementations lay the proper foundations for
+checkUp feature implementations. However, due to managing workload in a group setting and balancing
+reviewing teammate's PRs and writing our own code, the difficulty increased due to the sheer workload.
+
+### Challenges faced
+
+Challenges we faced along the way include:
+
+1. Time management
+   - We found ourselves rushing most of the milestone deadlines, despite splitting it up to halves
+   - Last minute changes to the UI and features resulted in merge conflicts
+2. Team coordination
+   - All of us had differing schedules, therefore finding a common meeting time every week was difficult
+
+### Effort required
+
+Here are some of the efforts we have put in to develop checkUp, which was a brownfield project extended from AB3.
+
+1. Getting familiar with the large codebase.
+
+2. We have refactored the AB3 codebase to fit the needs of our application. This includes
+  renaming of classes, methods and variables to fit the context of our application.
+   - Implementing additional details to the `Person` class means refactoring of multiple other classes which
+       involve the `Person` class.
+   - Refactoring of test cases were also required to include the new details of the `Person` class.
+
+### Achievements
+
+Here are some of the features we achieved:
+
+1. Past and future appointments for patients
+2. Inpatient and outpatient patient types
+3. Filtering of patients according to given prefix
+4. Next of kin of patients
+5. Long-term medication of patients
```

Comment:
Also if it is missing an EOF whitespace line better to add 1 just in case

Reply:
Yeap thanks for the reminder! Confirmed there is an EOF whitespace.